services/order.py

```diff
@@ -0,0 +1,30 @@
+from django.db import transaction
+from datetime import datetime
+from db.models import Order, Ticket, User
+from db.models import MovieSession
+
+
+def create_order(tickets: list, username: str, date: str = None) -> User:
+    user = User.objects.get(username=username)
+
+    order_date = datetime.fromisoformat(date) if date else datetime.now()

```

Comment:
Ensure that the date string is validated to be in ISO format before using `datetime.fromisoformat(date)` to avoid potential errors.